src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private static int taskCount = 0;
+
+    public Task(String desc) {
+        this.description = desc;
+        this.isDone = false;
+        taskCount++;
+    }
+
+    public void setDone(boolean done) {
+        this.isDone = done;
+    }
```

Comment:
Not necessary to include "this" since your argument does not have the same name as the variables.